lib/ui/geometry.dart

```diff
@@ -329,43 +327,43 @@ class Offset extends OffsetBase {
 
   /// Compares two Offsets for equality.
   @override
-  bool operator ==(dynamic other) {
+  bool/*!*/ operator ==(dynamic other) {
     return other is Offset
         && other.dx == dx
         && other.dy == dy;
   }
 
   @override
-  int get hashCode => hashValues(dx, dy);
+  int/*!*/ get hashCode => hashValues(dx, dy);
 
   @override
-  String toString() => 'Offset(${dx?.toStringAsFixed(1)}, ${dy?.toStringAsFixed(1)})';
+  String/*!*/ toString() => 'Offset(${dx?.toStringAsFixed(1)}, ${dy?.toStringAsFixed(1)})';
```

Comment:
Kill the `?`s here.

Reply:
The tool actually did it automatically! But removed anyway.